tests/e2e/tests/test_accommodation_v2.go

```diff
@@ -0,0 +1,465 @@
+func TestAccommodationProductListServiceV2WithFilter(t *testing.T, tt *Test, distributorBot *bot.Bot, supplierBot *bot.Bot, ctx context.Context) {
+	// Modification timestamp which should exactly return one result (see hotelCode).
+	// See the properties.json file in the pp-mock for more info
+	const modifiedAfterSecs int64 = 1710489050
+	const hotelCode = "HOTEL567890"
+
+	resp, err := distributorBot.AccommodationProductListServiceV2.AccommodationProductList(
+		requestContext(ctx, &metadata.Metadata{
+			Recipient: supplierBot.CMAccountAddress().Hex(),
+		}),
+		&accommodationv2.AccommodationProductListRequest{
+			Header: &typesv1.RequestHeader{BaseHeader: &typesv1.Header{}},
+			ModifiedAfter: &timestamppb.Timestamp{
+				Seconds: modifiedAfterSecs,
+			},
+		},
+	)
+	require.NoError(t, err)
+
+	tt.logger.Debug("AccommodationProductListServiceV2.AccommodationProductList response:\n", protoMessageToJSON(tt, resp))
+
+	require.Equal(t, typesv1.StatusType_STATUS_TYPE_SUCCESS, resp.Header.Status, "unexpected response status")
+	require.Empty(t, resp.Header.Alerts, "unexpected response alerts")
+
+	// The response should contain only one property as only one is modified after the given timestamp
+	require.Len(t, resp.Properties, 1, "unexpected number of properties in response")
+
+	// Let's check if result is as expected
+	require.NotEmpty(t, resp.Properties, "unexpected empty response properties")
+	require.NotEmpty(t, resp.Properties[0].SupplierCode, "unexpected empty response properties[0].SupplierCode")
+	require.NotEmpty(t, resp.Properties[0].SupplierCode.SupplierCode, "unexpected empty response properties[0].SupplierCode.SupplierCode")
+	require.Equal(t, hotelCode, resp.Properties[0].SupplierCode.SupplierCode, "unexpected response properties[0].SupplierCode.SupplierCode")
+}
+
+/* Get detailed accommodation information for a specific hotel code (supplier code). */
+func TestAccommodationProductInfoServiceV2(t *testing.T, tt *Test, distributorBot *bot.Bot, supplierBot *bot.Bot, ctx context.Context) {
+	const hotelCode = "HOTEL789012"
+
+	resp, err := distributorBot.AccommodationProductInfoServiceV2.AccommodationProductInfo(
+		requestContext(ctx, &metadata.Metadata{
+			Recipient: supplierBot.CMAccountAddress().Hex(),
+		}),
+		&accommodationv2.AccommodationProductInfoRequest{
+			Header: &typesv1.RequestHeader{BaseHeader: &typesv1.Header{}},
+			SupplierCodes: []*typesv2.SupplierProductCode{
+				{SupplierCode: hotelCode},
+			},
+		},
+	)
+	require.NoError(t, err)
+
+	tt.logger.Debug("AccommodationProductInfoServiceV2.AccommodationProductInfo response:\n", protoMessageToJSON(tt, resp))
+
+	require.Equal(t, typesv1.StatusType_STATUS_TYPE_SUCCESS, resp.Header.Status, "unexpected response status")
+	require.Empty(t, resp.Header.Alerts, "unexpected response alerts")
+
+	// The response should contain only the one property filtered in the request
+	require.NotEmpty(t, resp.Properties, "unexpected empty response properties")
```

Comment:
redundant, imo
look for similar redundant cases, there are others

Reply:
Removed